kafka/config.go

```diff
@@ -31,10 +31,10 @@ func WithMessageHandlers(messageHandlers ...MessageHandler) Option {
 	}
 }
 
-// WithDebug enables debug.
-func WithDebug() Option {
+// WithDebug enables/disables debug.
+func WithDebug(enabled bool) Option {
```

Comment:
```suggestion
func WithDebug(flag bool) Option {
```

`flag` name is better that enables if you can enable/disable.

Reply:
I rather prefer to read the whole function as: `With Debug Enabled: True` / `With Debug Enabled: False`
The term `flag` is very tied to system flags as arguments.

Reply:
Ok, my fault.

Reply:
No fault at all! haha you made a really good point.